bim2sim/tasks/bps/bind_tz.py

```diff
@@ -17,101 +17,87 @@ class CombineThermalZones(ITask):
         * Usage
         * Window to wall ratio
     """
-
-    # for 1Zone Building - workflow.zoning_setup: LOD.low -
-    # Disaggregations not necessary
     reads = ('elements',)
-    touches = ('bounded_tz',)
-
-    def __init__(self, playground):
-        super().__init__(playground)
-        self.bounded_tz = []
 
     def run(self, elements):
-        tz_elements = filter_elements(elements, 'ThermalZone', True)
+        tz_elements = filter_elements(elements, 'ThermalZone')
         n_zones_before = len(tz_elements)
-        self.logger.info("Try to reduce number of thermal zones by merging")
+        self.logger.info("Try to reduce number of thermal zones by combining.")
         if len(tz_elements) == 0:
             self.logger.warning("Found no spaces to bind")
         else:
             if self.playground.sim_settings.zoning_setup is LOD.low:
                 self.bind_tz_one_zone(
-                    list(tz_elements.values()), elements)
+                    tz_elements, elements)
             elif self.playground.sim_settings.zoning_setup is LOD.medium:
-                self.bind_tz_criteria(elements)
-            else:
-                self.bounded_tz = list(tz_elements.values())
+                self.bind_tz_criteria(tz_elements, elements)
+            tz_elements_after = filter_elements(
+                elements, 'ThermalZone')
             self.logger.info("Reduced number of thermal zones from %d to  %d",
-                             n_zones_before, len(self.bounded_tz))
-        self.add_storeys_to_buildings(elements)
+                             n_zones_before, len(tz_elements_after))
 
-        return self.bounded_tz,
-
-    def bind_tz_one_zone(self, thermal_zones, elements):
+    @staticmethod
+    def bind_tz_one_zone(thermal_zones, elements):
```

Comment:
same here, please refactor these methods to something like "combine" / "aggregate" / ...

Reply:
Same for following methods